examples/https-nginx/README.md

```diff
@@ -137,36 +137,11 @@ $ curl https://104.198.1.26:30028 -k
 ...
 ```
 
-Then we will update the configmap. First check your kubectl version.
-
-```sh
-$ kubectl version 
-Client Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.4", GitCommit:"dd6b458ef8dbf24aff55795baa68f83383c9b3a9", GitTreeState:"clean", BuildDate:"2016-08-01T16:45:16Z", GoVersion:"go1.6.2", Compiler:"gc", Platform:"linux/amd64"}
-Server Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.5", GitCommit:"b0deb2eb8f4037421077f77cb163dbb4c0a2a9f5", GitTreeState:"clean", BuildDate:"2016-08-11T20:21:58Z", GoVersion:"go1.6.2", Compiler:"gc", Platform:"linux/amd64"}
-```
-
-If you are using 1.5 or higher:
-Edit file `default.conf`: change `index index.html;` in line 8 to `index index2.html;`.
-
-```sh
-$ kubectl replace configmap nginxconfigmap --from-file=default.conf
-configmap "nginxconfigmap" replaced
-```
-
-If you are using 1.4 or lower:
-Retrieve configmap nginxconfigmap.
-
-```sh
-$ kubectl get configmap nginxconfigmap -o yaml > examples/https-nginx/nginxcm.yaml
-```
-
-Edit file `nginxcm.yaml`: change `index index.html;` to `index index2.html;`.
-Apply the change.
-
+Then we will update the configmap.
 ```sh
-$ kubectl apply -f examples/https-nginx/nginxcm.yaml 
-configmap "nginxconfigmap" configured
+$ kubectl edit
```

Comment:
Yes, I have tried. `edit` works as expected.
Both `kubectl edit` and `kubectl apply` are use PATCH, and configmap is a map[string]string, which can be patched.
>that way you can show the user the entier thing, and they directly port this example to a shell script, for instances

What do you think? We use `kubectl apply`?